hexgame/tests/test_board.py

```diff
@@ -1,24 +1,25 @@
-from hexgame.src import board as Board
-from hexgame.src import cell as Cell
+from hexgame.src.board import Board 
```

Comment:
You are introducing a space at the end of the line here.

Reply:
yeah if this is a flake8 thing I am not noticing